### DISCUSSION TITLE
Missing/unlinked page from docs

### DISCUSSION THREAD

**Discussion by Kunz-David** [pmndrs/jotai#1859] [General]
Hi,

I was looking for some details about `splitAtom`, but didn't find the necessary details in the docs, so I used BingChat. I was surprised that it gave me the details I needed and provided a link to the docs, which I couldn't find on the jotai webpage.

Here is the page: [https://jotai.org/docs/utilities/split](https://jotai.org/docs/utilities/split). It should be in utilities but is not there. 

Since it was useful to me, do you think it should be linked in the docs? It is probably from [this PR](https://github.com/pmndrs/jotai/pull/1097)

Kind Regards,
David

**Comment by t6adev** [comment]
Oh, I didn't realize there is not in the utility menu.
The page is linked from [https://jotai.org/docs/recipes/large-objects](https://jotai.org/docs/recipes/large-objects).
There is [Call for document contributors](https://github.com/pmndrs/jotai/discussions/1627) discussion. Nice find!
